data/battle_scripts_1.s

```diff
@@ -471,16 +471,16 @@ BattleScript_OctolockEndTurn::
 	jumpifstat BS_TARGET, CMP_GREATER_THAN, STAT_SPDEF, MIN_STAT_STAGE, BattleScript_OctolockTryLowerSpDef
 	goto BattleScript_OctolockEnd2
 BattleScript_OctolockLowerDef:
-	playstatchangeanimation BS_ATTACKER, BIT_DEF | BIT_SPDEF, STAT_CHANGE_NEGATIVE
 	jumpifability BS_TARGET, ABILITY_BIG_PECKS, BattleScript_OctolockTryLowerSpDef
```

Comment:
big pecks is now lowering just sp def, but not playing any stat change animation. You can set `setbyte sSTAT_ANIM_PLAYED, TRUE` after the first one, and then check if this byte is set to decide whether or not to play just the defense-lowering anim

Reply:
Done! Ready for re-re-review!